includes/polling/functions.inc.php

```diff
@@ -489,7 +489,7 @@ function location_to_latlng($device)
     $device_location = $device['location'];
     if (!empty($device_location)) {
         $new_device_location = preg_replace("/ /", "+", $device_location);
-        $new_device_location = preg_replace('/[^A-Za-z0-9\-\+]/', '', $device_location); // Removes special chars.
+        $new_device_location = preg_replace('/[^A-Za-z0-9\-\+]/', '', $new_device_location); // Removes special chars.
```

Comment:
Is this right still? I'm not sure we should have merged the last one as we're basically replacing spaces with + and then getting rid of the + again.